site/content/blog/2025-08-01-zap-updates-july-2025/index.md

```diff
@@ -0,0 +1,234 @@
+
+## GitHub Pulse
+Here are some statistics for the two main ZAP repositories:
+
+[zaproxy](https://github.com/zaproxy/zaproxy/pulse/monthly)  
+Excluding merges, 6 authors have pushed 12 commits to main and 12 commits to all branches. On main, 42 files have changed and there have been 277 additions and 151 deletions.
+
+[zap-extensions](https://github.com/zaproxy/zap-extensions/pulse/monthly)  
+Excluding merges, 7 authors have pushed 88 commits to main and 92 commits to all branches. On main, 1,708 files have changed and there have been 72,176 additions and 3,535 deletions.
+
+A total of [87 human PRs were merged](https://github.com/search?q=org%3Azaproxy+type%3Apr+-author%3Azapbot+-author%3Aapp%2Fdependabot+sort%3Aupdated-asc+closed%3A2025-07+is%3Amerged&type=pullrequests) on the ZAP repos.
+
+## Released Add-ons - Full Changelog
+In July 2025, we released updated versions of 15 add-ons:
+
+#####  Passive scanner rules
+**v61.1.0**  
+Added
+- A ZAP is Out of Date rule.
+
+Changed
+- Updated help with specific Category identifiers for use with the Custom Payloads add-on for rules:
+    - Application Error Disclosure
+    - Information Disclosure - Suspicious Comments
+    - Username Hash Found
+
+##### Ajax Spider
+**v23.25.0**  
+Fixed
+- Correct configuration key for Logout Avoidance (Issue 8994).
+- Error logs to always include stack trace.
+
+##### Authentication Helper
+**v0.27.0**  
+Added
+- Support for recorded scripts in the Authentication Tester.
+
+Changed
+- Updated to depend on Zest add-on 48.8.0.
+
+##### Automation Framework
+**v0.51.0**  
+Added
+- Support for exclude regexes to active scan config job.
+- Always run option for all jobs.
+- Support for data driven nodes in plan (not yet in the UI).
+
+Changed
+- Job remains selected when moved in the GUI.
+
+##### Common Library
+**v1.34.0**  
+Added
+- Added Alert Tags for PCI DSS and HIPAA standards.
+- Added a help page for the alert tags provided through this add-on.
+
+##### DOM XSS Active scanner rule
+**v22**  
+Changed
+- Allow to use Edge.
+- Depend on newer version of Selenium add-on.
+- Maintenance changes.
+
+##### Linux WebDrivers
+**v150**  
+Changed
+- Update ChromeDriver to 138.0.7204.183.
+
+**v149**  
+Changed
+- Update ChromeDriver to 138.0.7204.168.
+
+**v148**  
+Changed
+- Update ChromeDriver to 138.0.7204.157.
+
+**v147**  
+Changed
+- Update ChromeDriver to 138.0.7204.94.
+
+**v146**  
+Changed
+- Update ChromeDriver to 138.0.7204.92.
+
+##### MacOS WebDrivers
+**v150**  
+Changed
+- Update ChromeDriver to 138.0.7204.183.
+
+**v149**  
+Changed
+- Update ChromeDriver to 138.0.7204.168.
+
+**v148**  
+Changed
+- Update ChromeDriver to 138.0.7204.157.
+
+**v147**  
+Changed
+- Update ChromeDriver to 138.0.7204.94.
+
+**v146**  
+Changed
+- Update ChromeDriver to 138.0.7204.92.
+
+##### Passive scanner rules
+**v66**  
+Added
+- The Reverse Tabnabbing and Retrieved from Cache scan rules now have CWE references.
+- A ZAP is Out of Date rule.
+
+**v57.1.0**  
```

Comment:
Do we need to indicate this was a back-port release? Where is/are the other(s).

Reply:
The other was at the top, it seems the script is not trimming the names (it has extra space).